draft-bastian-jose-dvs.md

```diff
@@ -92,15 +92,28 @@ This algorithm is intended for use with digital credentials ecosystems, includin
 
 {::boilerplate bcp14-tagged}
 
-# Terminology
+# The "pkds" Header Parameter
 
-The draft uses "JSON Web Signature", "JOSE Header", "JWS Signature", "JWS Signing Input" as defined by {{RFC7515}}.
+The pkds protected header parameter specifies the inputs needed to derive a symmetric key for MAC computation using a key agreement and derivation scheme. Its value is a JSON object that includes identifiers, public keys, and algorithm-specific parameters relevant to the derivation.
 
-Signing Party:
-: The Party that performs the key agreement first and generates the MAC. Similar to a Signer.
+## Syntax and semantics
 
-Verifying Party:
-: The Party that performs the key agreement second, generates the MAC and compares it to a given value. Similar to a Verifier.
+The `pkds` Header Parameter value MUST be a JSON object with the following fields:
+
+* `rpk` (object, REQUIRED): The Recipient's public key used in DH-KA. The `rpk` object MUST contain at least one key claim as defined in {{RFC7515 Section 4.1}}.
+
+  Implementations MUST reject a JWS if the `rpk` key cannot be resolved unambiguously at validation time.
+  
+* `ppk` (object, OPTIONAL):  The JWS Producer’s public key used in DH-KA. The `ppk` object MUST contain at least one key claim as defined in {{RFC7515 Section 4.1}}.
```

Comment:
Why is ppk OPTIONAL? I guess you are hinting that in SD-JWT case we would take this from "cnf"? Better make it RECOMMENDED unless known from context/other claims